### PR TITLE
Update how combined repos are defined in the README

### PR DESCRIPTION
**Pull request by andrewschoen**
Signed-off-by: Andrew Schoen <aschoen@redhat.com>

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -340,9 +336,10 @@ like 'radosgw-agent'. This inclusion would be defined at the project level, like
     repos = {
         'ceph': {
             'extras': ['ceph-deploy', 'radosgw-agent'],
-            'deb': {
```
Comment (alfredodeza) [inline README.rst]: I just confirmed that it is not possible to deal with combined repos for anything that is not a DEB. That would make this key redundant.
An alternative would be something like:

    'combined': ['wheezy', 'precise', 'jessie']

Comment (alfredodeza) [inline README.rst]: *however*, we could also keep `'combined': True` and make them *all* the ones available for the DEBs be combined.

When/why/where would it make sense to have certain distro versions combined and not the rest of them? With this key being a boolean is either you get them all combined or not.

Comment (andrewschoen) [inline README.rst]: If we remove the ``deb`` key what will the endpoint look like? We were going to use that key as the way to map the endpoint to this configuration.

Comment (alfredodeza) [inline README.rst]: we should default to something, like `multi`, `combined`, or `mixed`. I think having one of these will be better than to rely on always-correct configuration.

**Comment by andrewschoen** [comment]
@alfredodeza I've updated this, let me know what you think.

**Pull request merged**